packages/core/src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -362,22 +365,22 @@ export async function pollTransformationStatusUntilComplete(jobId: string) {
     try {
         status = await pollTransformationJob(jobId, CodeWhispererConstants.validStatesForCheckingDownloadUrl)
     } catch (error) {
-        const errorMessage = 'Failed to get transformation job status'
+        const errorMessage = CodeWhispererConstants.failedToCompleteJobMessage
         getLogger().error(`CodeTransformation: ${errorMessage}`, error)
         transformByQState.setJobFailureErrorMessage(errorMessage)
-        throw new ToolkitError(errorMessage, { cause: error as Error })
+        throw new Error('Poll job failed')
     }
 
     return status
 }
 
 export async function finalizeTransformationJob(status: string) {
     if (!(status === 'COMPLETED' || status === 'PARTIALLY_COMPLETED')) {
-        const errorMessage = 'Failed to complete transformation'
+        const errorMessage = CodeWhispererConstants.failedToCompleteJobMessage
         getLogger().error(`CodeTransformation: ${errorMessage}`)
         sessionPlanProgress['transformCode'] = StepProgress.Failed
         transformByQState.setJobFailureErrorMessage(errorMessage)
-        throw new ToolkitError(errorMessage, { code: 'JobDidNotSucceed' })
```

Comment:
Question: We are ditching the ToolkitError because that forces a notification to pop up?

What other side effects are there from removing ToolkitError? Any logging concerns?

Reply:
Correct that forces a notification to pop up. Not aware of any other side effects, we already handle notifications, telemetry, and logging ourselves everywhere.

Reply:
No major concerns with ditching the ToolkitError. The message the user sees will not be formatted as nicely but it will still be logged. For reference, [this handles displaying Errors](https://github.com/aws/aws-toolkit-vscode/blob/master/packages/core/src/extensionShared.ts#L203)

Reply:
I think a side effect is `telemetry_1.telemetry.ui_click.run(() => {` the `.run` telemetry event picks up `ToolkitError` and sets fields automatically.

We currently don't use `.run` in our product.